cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -18,6 +18,18 @@ spec:
     - rcc
   scope: Namespaced
   version: v1beta1
+  additionalPrinterColumns:
```

Comment:
Another helpful column is `.status.state`. The operator currently only reports `creating` or `created`, but this will improve over time.

Reply:
Just added `.status.state` and resolved the conflict.